Source/src/Gameplay.cpp

```diff
@@ -116,6 +117,11 @@ Hachiko::GameObject* Hachiko::SceneManagement::FindInCurrentScene(
     return App->scene_manager->GetRoot()->Find(id);
 }
 
+HACHIKO_API std::vector<Hachiko::GameObject*> Hachiko::SceneManagement::Instantiate(unsigned long long prefab_uid, GameObject* parent, unsigned n_instances)
+{
+    return App->resources->InstantiatePrefab(prefab_uid, parent, n_instances);
+}
+
```

Comment:
Maybe we can make this a `static` method of `GameObject` just like other `Instantiate` methods. What do you think?

Reply:
It was not letting me do it an std vector return thats why i changed it :(